components/Goals.vue

```diff
@@ -19,14 +23,13 @@
             title="YouTube video player"
             allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
             allowfullscreen
-          >
-          </iframe>
```

Comment:
Where was this iframe opened, if it's deleted from here, why isn't the opening tag deleted (its not showing up in the differences)